client/blocks/credit-card-form/index.jsx

```diff
@@ -51,7 +51,22 @@ const CreditCardForm = createReactClass( {
 	},
 
 	_mounted: false,
-	fieldNames: [ 'name', 'number', 'cvv', 'expirationDate', 'country', 'postalCode' ],
+	fieldNames: [
+		'name',
+		'number',
+		'cvv',
+		'expirationDate',
+		'country',
+		'postalCode',
+		'streetNumber',
+		'address1',
```

Comment:
Should address fields have the dash or not? In any case, I think un underscore is better that a dash :)

Reply:
The general pattern across these forms, at least from what I've seen is to use camelCase when interacting with `formState`, kebab-case for the field names, and snake_case for the props sent to the API.

I'm not certain if this is this the most optimal approach, but it's pretty well [cooked](https://github.com/Automattic/wp-calypso/blob/1a6f0e0a9bf53ae00c296e0733a450d7d1411858/client/blocks/credit-card-form/index.jsx#L245)  into the [code](https://github.com/Automattic/wp-calypso/blob/1a6f0e0a9bf53ae00c296e0733a450d7d1411858/client/blocks/credit-card-form/index.jsx#L254).

It looks like some remodelling is being done in the [transaction store](https://github.com/Automattic/wp-calypso/blob/1a6f0e0a9bf53ae00c296e0733a450d7d1411858/client/lib/store-transactions/index.js#L299) anyway.